operations/pyroscope/helm/pyroscope/templates/_helpers.tpl

```diff
@@ -91,3 +91,16 @@ If no components are defined fall back to single binary statefulset
 all: {kind: "StatefulSet", name: "{{$full_name}}"}
 {{- end }}
 {{- end }}
+
+
+{{- define "cluster.domain" }}
```

Comment:
I am not too sure at why we require those replace statements.

In my opinion a normal values.yaml variable should be enough to allow people to override it.